docs/decisions/0005-conditional-import-of-confluent-kafka.rst

```diff
@@ -0,0 +1,43 @@
+4. Conditional import of confluent-kafka
+=======================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+* confluent-kafka is a library written and distributed by Confluent, our managed instance provider. It abstracts out
+the work for sending and receiving events to and from the Kafka cluster and converting them into message objects.
+* confluent-kafka relies on a C library called librdkafka-dev
+* librdkafka-dev does not currently have a compiled binary available for Linux/aarch64
```

Comment:
spelling typo: `aarch64`

Reply:
I think that's actually the correct spelling https://archlinuxarm.org/packages/aarch64/linux-aarch64